UserProfile.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameFace.Struct
+{
+    public static class Helper
+    {
+        public static List<SprintRecords> DevideBySprints(List<UserRecords> list)
+        {
+            DateTime date = list[0].Date;
+            int quarterNumber = (date.Month - 1) / 3 + 1;
+            DateTime firstDayOfQuarter = new DateTime(date.Year, (quarterNumber - 1) * 3 + 1, 1);
+            DateTime lastDayOfQuarter = firstDayOfQuarter.AddMonths(3).AddDays(-1);
```

Comment:
It is not necessary, but if we wanted to be fancy we could use some extension methods here. It is just syntax sugar, but it does have the benefit that the intention of the code is a bit clearer, and it can be used by any DateTime in the project. An example of what it might look like if we choose to use extension methods:

```csharp
int quarterNumber = date.QuarterNumber();
int firstDayOfQuarter = date.FirstDayOfQuarter();
int lastDayOfQuarter = date.LastDayOfQuarter();
```
Since this encapsulates the _how_  from the _what_ and allows slightly better reuse.